tests/binaries/808.sol

```diff
@@ -0,0 +1,8 @@
+library X {
```

Comment:
can we rename this file to have a more descriptive name? the number scheme was just for contracts that correspond to github issues.